src/ofxGLWarper.h

```diff
@@ -16,7 +16,7 @@
 class ofxGLWarper{
 	
 public:
-    enum CornerLocation{
+    enum CornerID{
```

Comment:
I would not change this as it does not provide any improvement and it introduces backwards compatibility issues.